deployment/copilot/README.md

```diff
@@ -117,28 +119,26 @@ copilot env deploy --name dev
 copilot svc deploy --name traffic-comparator-jupyter --env dev
 copilot svc deploy --name traffic-comparator --env dev
 copilot svc deploy --name traffic-replayer --env dev
-
-copilot svc deploy --name elasticsearch --env dev
-copilot svc deploy --name capture-proxy --env dev
-copilot svc deploy --name opensearch-benchmark --env dev
+copilot svc deploy --name capture-proxy-es --env dev
+copilot svc deploy --name migration-console --env dev
 ```
 
 ### Running Benchmarks on the Deployed Solution
 
-Once the solution is deployed, the easiest way to test the solution is to exec into the benchmark container and run a benchmark test through, as the following steps illustrate
+Once the solution is deployed, the easiest way to test the solution is to exec into the migration-console container and run a benchmark test through, as the following steps illustrate
 
 ```
 // Exec into container
-copilot svc exec -a migration-copilot -e dev -n opensearch-benchmark -c "bash"
+copilot svc exec -a migration-copilot -e dev -n migration-console -c "bash"
```

Comment:
Ah, this resolves a comment I made at the very beginning about having an example of this command! Feel free to disregard!